solutions/banking/main.tf

```diff
@@ -188,28 +188,21 @@ resource "shell_script" "watson_assistant" {
     create = file("${path.module}/watson-scripts/assistant-create.sh")
     delete = file("${path.module}/watson-scripts/assistant-destroy.sh")
     read   = file("${path.module}/watson-scripts/assistant-read.sh")
-    update = file("${path.module}/watson-scripts/assistant-create.sh")
   }
 
   environment = {
-    WATSON_ASSISTANT_URL = local.watsonx_assistant_url
+    WATSON_ASSISTANT_API_VERSION = "2023-06-15"
+    WATSON_ASSISTANT_DESCRIPTION = "Generative AI sample app assistant"
+    WATSON_ASSISTANT_LANGUAGE    = "en"
+    WATSON_ASSISTANT_NAME        = "gen-ai-rag-sample-app-assistant"
+    WATSON_ASSISTANT_URL         = local.watsonx_assistant_url
   }
 
   sensitive_environment = {
     API_KEY = var.ibmcloud_api_key
```

Comment:
It may be better to name it IBMCLOUD_API_KEY - that might help in case you will use ibmcloud CLI - then it will pick it up automatically